tests/benchmarks/invalid_benchmark.py

```diff
@@ -0,0 +1,2 @@
+def get_benchmark():
```

Comment:
The tests are inside the package now as of a week or two ago, so I'm not sure these are even running? Can you move them there and/or double check.

Reply:
I am not sure I get it. Could you please elaborate it further ?

Reply:
These files belong in `bowtie/tests/benchmarks`, right now they probably don't even run in CI at all.

Reply:
`bowtie/tests/test_benchmarks.py` is loading the file, so it is being used right? Or else the tests would also fail.

Reply:
I'm looking at the GitHub UI not the git branch, so maybe you're saying the UI is wrong, but `bowtie/tests/test_benchmarks.py` doesn't exist, you have `tests/test_benchmarks.py`, which I again suspect isn't running at all, now that the tests are at `bowtie/tests`!

Reply:
Oh my bad :( I didnt merge the main branch and didnt realize the location of tests has changed. Will change that